pkg/services/ngalert/api/tooling/definitions/ruler_state_history.go

```diff
@@ -12,6 +12,8 @@ import "github.com/grafana/grafana-plugin-sdk-go/data"
 //     Responses:
 //       200: StateHistory
 
+// swagger:response StateHistory
```

Comment:
This was not strictly related to this PR, but figured I'd fix the error in the swagger spec while regenerating.